src/users/cd-tickets.vue

```diff
@@ -39,10 +41,21 @@
         this.currentUser = res.body.user;
       },
       async loadEvents() {
-        this.usersDojos.forEach(async (dojo) => {
-          const res = await EventService.loadEvents(dojo.dojoId);
-          this.events = this.events.concat(res.body);
-        }, this);
+        const query = { status: 'published' };
+        query.afterDate = moment().unix();
+        query.utcOffset = moment().utcOffset();
+
+        return Promise.all(this.usersDojos.map(dojo =>
+          EventService.v3.get(dojo.dojoId, {
+            params: {
+              query,
+              related: 'sessions.tickets',
+            } })))
+          .then((events) => {
+            this.events = (events.reduce((red, dojoEvents) =>
+              red.concat(dojoEvents.body.results), []))
+            .sort(EventUtils.orderByStartTime);
```

Comment:
We don't need this sort anymore, do we? Now that it's handled on the backend.

Reply:
We do. It's an aggregation of all Dojos events of which the users belongs to